src/main/java/com/nerodesk/om/mock/MkFriends.java

```diff
@@ -83,16 +97,19 @@ public boolean leader() throws IOException {
         assert this.dir != null;
         assert this.user != null;
         assert this.label != null;
```

Comment:
@alevohin ok. if I remove them I get PDD complaining about those variables not being used to they should become local variables... what do you suggest?